composer.json

```
@@ -7,8 +7,8 @@
     "php": ">=5.6",
     "composer/installers": "^v1.12.0 || ^2.2",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7 || ^1.0",
-    "wp-coding-standards/wpcs": "dev-develop#fca9d9e as 2.3.0",
-    "automattic/vipwpcs": "2.3.3",
+    "wp-coding-standards/wpcs": "dev-develop#bb792cb as 2.3.0",
+    "automattic/vipwpcs": "^2.3.3 || ^2.3.4",
```

Why 2 versions?

Given the desired update to WPCS 3.0 I'd probably just switch to dev-develop here as a new release hasn't been tagged yet.

```suggestion
    "automattic/vipwpcs": "dev-develop",
```